target/linux/realtek/files-5.10/arch/mips/rtl838x/setup.c

```diff
@@ -50,24 +91,19 @@ void __init plat_mem_setup(void)
 
 void __init plat_time_init(void)
 {
-	struct device_node *np;
-	u32 freq = 500000000;
+	u32 cpu, lxb, mem;
 
 	of_clk_init(NULL);
 	timer_probe();
 
-	np = of_find_node_by_name(NULL, "cpus");
-	if (!np) {
-		pr_err("Missing 'cpus' DT node, using default frequency.");
-	} else {
-		if (of_property_read_u32(np, "frequency", &freq) < 0)
-			pr_err("No 'frequency' property in DT, using default.");
-		else
-			pr_info("CPU frequency from device tree: %dMHz", freq / 1000000);
-		of_node_put(np);
-	}
+	plat_get_frequencies(&cpu, &lxb, &mem);
+	if (lxb && mem)
+		pr_info("Frequencies: CPU:%dMHz LXB:%dMHz MEM:%dMHz\n",
```

Comment:
```suggestion
		pr_info("Frequencies: CPU: %d MHz LXB: %d MHz MEM: %d MHz\n",
```
makes it human readable :p

Reply:
rewritten